playbook.md

```diff
@@ -673,6 +673,68 @@ Findings and recommendations are not the same. Findings mean that ‘we know thi
+
+Product managers need to be able to balance consider user research insights with constraints like time, capacity, and strategic objectives to prioritise the next most important thing. 
+
+Product managers help the team to work together, across disciplines, to identify potential solutions before choosing a way forward. 
+
+**Have a service mindset**
+Product managers recognise that a digital product is often not the whole solution to a problem, and that the people operating a new service day to day are users too.
+
+We think about the processes and systems around a product, including how those might change. 
+
+**Communicate clearly how the short term outcomes contribute to long term goals**
+Product managers need to own and articulate a vision that is consistent and meaningful to everyone with an interest in the project, from the delivery team all the way through to the most senior stakeholders, and users. 
+
+**Be present, collaborative, and consistent**
+Being available to the team to discuss the unexpected and share in the decision making is central to building empowered and protected teams. 
+
+Product Managers should regularly and actively communicate, review and defend our decisions - especially the unplanned or unexpected ones. 
+
+**Maintain a focus on outcomes**
+In the midst of delivery, key dates can loom large, and teams can be diverted onto things which seem urgent but aren’t important. 
+
+Product managers must help teams and stakeholders remain focused on outcomes over outputs, facilitating discussion and making space for reflection. 
+
+**Lead without authority**
+Product managers shouldn’t be bottlenecks. Product managers and delivery leads work together to build skilled multidisciplinary teams and create the space in which those disciplines can do their best work. 
+
+Product managers are open to new tools, techniques and approaches, working with teams to find what works.
+
+**Take blame, give credit away**
+Good product managers leave their ego at the door, taking negative feedback on board whilst ensuring the team are rightly recognised for their work and achievements. 
+
+Being kind is more important than being right - it’s a good rule of thumb to enter every conversation without assuming that you already know most, or best. 
+
+**Strong opinions, softly held**
+Product managers make the best decisions based on the available information with confidence, staying open to the possibility that things could change when more information is available in the future.
+
+Product managers communicate the trade-offs they’re making and the rationale for their decisions so that teams can understand how a decision was reached, even when they don’t agree. 
+
+**Prioritise** 
+Product managers work with the team, stakeholders and users to define priorities, and ensure these are clear. 
+Measure 
```

Comment:
Was this meant to be a section title, like the bolded "**Prioritise**" above? In that case it would look better with a blank line separating it from the section above.